example/objective_c/README.md

```diff
@@ -0,0 +1,36 @@
+# Objective C example
+
+This example shows how to use ffigen to generate bindings for an Objective C
+library. It uses the AVFAudio framework to play audio files.
+
+```
+dart play_audio.dart test.mp3
+```
+
+## Config notes
+
+The ffigen config for an Objective C library looks very similar to a C library.
+The most important difference is that you must set `language: objc`. If you want
+to filter which interfaces are included you can use the `objc-interfaces:`
+option. This works similarly to the other filtering options.
+
+It is recommended that you filter out just about everything you're not
+interested in binding (see the ffigen config in [pubspec.yaml](./pubspec.yaml)).
+Virtually all Objective C libraries depend on Apple's internal libraries, which
+are huge. Filtering can reduce the generated bindings from millions of lines to
+tens of thousands.

```

Comment:
The generated file is still 60000 LOC. This is not really a problem, but should we consider being able to filter members inside objective c interfaces?

I've filed an issue to track it.

* https://github.com/dart-lang/ffigen/issues/419

Reply:
Yeah, I have a note about that somewhere on my todo list.

Reply:
Feel free to land it with the 60000 LOC.

(Fun fact: the 60000 LOC file is bigger than the mp3. 😅)